frontera.md

```diff
@@ -27,10 +35,22 @@ To run the TPV13 scenario, you should:
 
 ```
 cd seissol-training/tpv13
-mpirun singularity run ~/my-training.sif gmsh -3 tpv13_training.geo
-mpirun singularity run ~/my-training.sif pumgen -s msh2 tpv13_training.msh
-OMP_NUM_THREADS=28 mpirun -n 2 singularity run ~/my-training.sif seissol parameters.par
+mpirun apptainer run ~/my-training.sif gmsh -3 tpv13_training.geo
+mpirun apptainer run ~/my-training.sif pumgen -s msh2 tpv13_training.msh
+OMP_NUM_THREADS=28 mpirun -n 2 apptainer run ~/my-training.sif seissol parameters.par
```

Comment:
Either we need to run with:
`SEISSOL_COMMTHREAD=0 OMP_NUM_THREADS=28 mpirun -n 2 apptainer run ~/my-training.sif seissol parameters.par`
or 
`OMP_NUM_THREADS=26 mpirun -n 2 apptainer run ~/my-training.sif seissol parameters.par`
or
`OMP_NUM_THREADS=47 apptainer run ~/my-training.sif seissol parameters.par`

Else I get the error:
>There are no free CPUs left.

Reply:
best seems to be 
`OMP_NUM_THREADS=26 mpirun -n 2 apptainer run ~/my-training.sif seissol parameters.par`

Reply:
I tested it just now. OMP_NUM_THREADS=28 worked for me. Do you run directly on Frontera or you used the jupyter notebook?

Reply:
I run directly on Frontera, on an interactive node.

Reply:
Actually, it might be that you use an older docker version:
Check that you get:
```
Tue May 21 08:32:38, Info:  Welcome to SeisSol 
Tue May 21 08:32:38, Info:  Copyright (c) 2012 - 2024  SeisSol Group 
Tue May 21 08:32:38, Info:  Version: v1.1.3 
```